### ISSUE TITLE
rebar3_lfe plugin failed to compile on Erlang/OTP 26 due to a a failed call to user:start() in lfe_init

### ISSUE THREAD

**Issue by tj800x** [lfe/rebar3#80]
In order to get the plugin to compile correctly I had to comment out the user:start() call below:
```
start() ->
    case collect_args(init:get_plain_arguments()) of
        {[],[]} ->                              %Run a shell
            user_drv:start(['tty_sl -c -e',{lfe_shell,start,[]}]);
        {Es,Script} ->
            % user:start(),
            %% io:format("es: ~p\n", [{Es,Script}]),
            run_evals_script(Es, Script)
    end. (edited) 
```

Related tickets:
* #79 
* https://github.com/lfe/lfe/issues/474

**Comment by oubiwann** [comment]
This is an upstream issue and has been addressed in LFE 2.1.2 (see https://github.com/lfe/lfe/releases/tag/v2.1.2).

I've got a couple of rebar3_lfe releases planned, the first of which will include an update to the latest LFE.

**Comment by oubiwann** [comment]
With LFE 2.1.2 and rebar3 3.22.1, this particular issue is solved.
